virtualcluster/cmd/syncer/app/server.go

```diff
@@ -132,6 +132,22 @@ func Run(cc *syncerconfig.CompletedConfig, stopCh <-chan struct{}) error {
 		}
 	}()
 
+	go func() {
+		// start a pprof http server
+		klog.Fatal(http.ListenAndServe(":6060", nil))
```

Comment:
I don't think this is an issue caused in this PR, but what area of code actually registers pprof endpoints with this HTTP server/muxer? 🤔

Reply:
This is the pprof behaviour - it handles the default muxer: https://pkg.go.dev/net/http/pprof

Reply:
Nice! Thanks 😄